shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/test/java/org/apache/shardingsphere/sql/parser/integrate/jaxb/domain/segment/impl/table/ExpectedTableFactor.java

```diff
@@ -39,6 +41,9 @@
     @XmlElement
     private ExpectedSubquery subquery;
     
+    @XmlElement
+    private List<ExpectedColumn> columns;
```

Comment:
You had better add at least one SQL case for `columns`.

Reply:
Columns have been deleted, and they will be added later with the test case